Integrations/EclecticIQ/EclecticIQ.py

```diff
@@ -0,0 +1,640 @@
+            }
+
+        standard_email_outputs.append(standard_email_output)
+
+    dbot_output = {
+        'Type': 'email',
+        'Indicator': email,
+        'Vendor': 'EclecticIQ',
+        'Score': score
+    }
+
+    context = {
+        'DBotScore': dbot_output
+    }
+
+    if observables:
+        human_readable_title = 'EclecticIQ Email reputation - {}'.format(email)
+        human_readable = tableToMarkdown(human_readable_title, integration_outputs)
+        context['EclecticIQ.Email(val.ID===obj.ID)'] = integration_outputs
+        context[outputPaths['email']] = standard_email_outputs
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': response,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': human_readable,
+        'EntryContext': context
+    })
+
+
+def domain_command():
+    domain = demisto.args()['domain']
+
+    response = get_observable(domain)
+
+    if 'total_count' in response and response['total_count'] == 0:
+        human_readable = 'No results found.'
+
+    integration_outputs = []
+    standard_domain_outputs = []
+
+    observables = response.get('data')
+
+    score = 0
+
+    for observable in observables:
+        meta = observable.get('meta', {})
+        maliciousness = meta.get('maliciousness')
+        score = maliciousness_to_dbotscore(maliciousness, DOMAIN_THRESHOLD)
+
+        integration_outputs.append({
+            'Name': domain,
+            'Created': observable.get('created_at'),
+            'LastUpdated': observable.get('last_updated_at'),
+            'ID': observable.get('id'),
+            'Maliciousness': maliciousness
+        })
+
+        standard_email_output = {
+            'Name': domain
+        }
+        if score == 3:
+            standard_email_output['Malicious'] = {
+                'Vendor': 'EclectiqIQ',
+                'Description': 'EclectiqIQ maliciousness confidence level: ' + maliciousness
+            }
+
+        standard_domain_outputs.append(standard_email_output)
+
+    dbot_output = {
+        'Type': 'domain',
+        'Indicator': domain,
+        'Vendor': 'EclecticIQ',
+        'Score': score
+    }
+
+    context = {
+        'DBotScore': dbot_output
+    }
+
+    if observables:
+        human_readable_title = 'EclecticIQ Domain reputation - {}'.format(domain)
+        human_readable = tableToMarkdown(human_readable_title, integration_outputs)
+        context['EclecticIQ.Domain(val.ID===obj.ID)'] = integration_outputs
+        context[outputPaths['domain']] = standard_domain_outputs
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': response,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': human_readable,
+        'EntryContext': context
+    })
+
+
+def get_observable(ioc):
+    cmd_url = '/api/observables?filter[value]={}'.format(ioc)
```

Comment:
"filter[value]" should be separated from the url and passed to request as params.
this way requests module will handle the URL escaping of special chars

Reply:
tried it, their api keep returning 401 Unauthorized for any request with `params`
will check it with them